cmd/contour/contour.go

```diff
@@ -165,12 +168,15 @@ func main() {
 
 		client, contourClient := newClient(*kubeconfig, *inCluster)
 
-		wl := log.WithField("context", "watch")
-		k8s.WatchServices(&g, client, wl, &reh)
-		k8s.WatchIngress(&g, client, wl, &reh)
-		k8s.WatchSecrets(&g, client, wl, &reh)
-		k8s.WatchIngressRoutes(&g, contourClient, wl, &reh)
-		k8s.WatchTLSCertificateDelegations(&g, contourClient, wl, &reh)
+		// resync timer disabled for Contour
+		coreInformers := coreinformers.NewSharedInformerFactory(client, time.Duration(0))
```

Comment:
you don't need to convert 0 to time.Duration, the literal constant is automatically converted to the type required by the function call.